modules/components/src/Table/helpers/columns.tsx

```diff
@@ -1,190 +1,204 @@
-import { css } from '@emotion/react';
-import { createColumnHelper } from '@tanstack/react-table';
-import { mergeWith } from 'lodash-es';
-import { type HTMLAttributes, useEffect, useRef } from 'react';
+import { css } from "@emotion/react";
```

Comment:
🙅 this is reverting to doublequotes, which tells me your prettier configs have an issue